### PR TITLE
Added banner

### PR DESCRIPTION
**Pull request by lukaszklekowski**

### CODE REVIEW

File: cardstack/header.php [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<?php
+require_once( __DIR__ . '/functions.php');
```
Comment (owoc) [inline cardstack/header.php]: Is this needed? Check at what stage functions.php is executed in WordPress. The class may already be available in this scope.

File: cardstack/templates/page-welcome.php [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        <?php endif; ?>
+
+    </article>
+
+    <article class="page<?php
+    if (get_theme_mod('page_breadcrumbs')) {
+        print(' has-breadcrumbs');
+    }
+    ?>">
+        <div class="container">
```
Comment (owoc) [inline cardstack/templates/page-welcome.php]: Should use sematic elements, not divs – maybe figure, figcaption…

File: cardstack/templates/page-welcome.php [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris semper dapibus eros.
+                    Quisque et rutrum purus, semper rhoncus purus. Donec libero odio, eleifend a purus
+                    lobortis, accumsan cursus urna. Pellentesque et pharetra metus. Mauris lobortis
+                    metus eu tortor blandit, in viverra est ultricies.
+                </p>
+            </div>
+        </div>
+    </article>
+    <aside style="display: flex; justify-content: space-between; margin: 0 auto 40px auto; max-width: 960px; box-shadow: 0 2px 1px #ccc; background-color: white;">
+        <div style="flex-basis: 33%; text-align: center; padding: 10px 0 0 0;">
```
Comment (owoc) [inline cardstack/templates/page-welcome.php]: Should be a list, not divs (similar to how the menu is rendered in this theme, so maybe even reuse styles).

File: cardstack/templates/page-welcome.php [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+    if (get_theme_mod('page_breadcrumbs')) {
+        print(' has-breadcrumbs');
+    }
+    ?>">
+        <div class="container">
+            <?php
+            print("<img src=".get_template_directory_uri()."/templates/Elementario.png alt=\"Amagi\"/>")
+            ?>
+            <div class="image-text">
+                <p>
```
Comment (owoc) [inline cardstack/templates/page-welcome.php]: Layout seems to break on smaller screens, when the text doesn't fit in the image. Text should be clipped in such a case.

**Review by owoc — CHANGES_REQUESTED** [review]

File: cardstack/templates/page-welcome.php [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    </article>
+
+    <article class="page<?php
+    if (get_theme_mod('page_breadcrumbs')) {
+        print(' has-breadcrumbs');
+    }
+    ?>">
+        <div class="container">
+            <?php
+            print("<img src=".get_template_directory_uri()."/templates/Elementario.png alt=\"Amagi\"/>")
```
Comment (owoc) [inline cardstack/templates/page-welcome.php]: Hardcode to https://static.animagia.pl/Amagi1.jpg so there's no need to store it under templates.

File: cardstack/templates/page-welcome.php [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+                    echo '</p>';
+                } else {
+                    echo '<h1>';
+                    the_title();
+                    echo '</h1>';
+                }
+                ?>
+
+
+                <?php
```
Comment (owoc) [inline cardstack/templates/page-welcome.php:57]: Please extract method printPremiumFilmPlayer and put it next to printFreeFilmPlayer (same code is also in template-premium-film).

File: cardstack/header.php (hunk 1 above)
Comment (lukaszklekowski) [reply cardstack/header.php]: Done

File: cardstack/templates/page-welcome.php (hunk 2 above)
Comment (lukaszklekowski) [reply cardstack/templates/page-welcome.php]: Done

File: cardstack/templates/page-welcome.php (hunk 3 above)
Comment (lukaszklekowski) [reply cardstack/templates/page-welcome.php]: Done

File: cardstack/templates/page-welcome.php (hunk 4 above)
Comment (lukaszklekowski) [reply cardstack/templates/page-welcome.php]: Done

File: cardstack/templates/page-welcome.php (hunk 5 above)
Comment (lukaszklekowski) [reply cardstack/templates/page-welcome.php]: Done

File: cardstack/templates/page-welcome.php (hunk 6 above)
Comment (lukaszklekowski) [reply cardstack/templates/page-welcome.php:57]: Done

**Pull request merged**